src/js/components/SchemaField.js

```diff
@@ -319,17 +263,28 @@ class SchemaField extends Component {
   }
 
   getFieldContent(errorMessage, autofocus) {
-    const { schema } = this.props;
+    const { schema, required, name } = this.props;
     const RADIO_SELECT_THRESHOLD = 4;
 
+    if (schema.media) {
+      return (
+        <MountService.Mount
+          type={`SchemaField:${schema.media.type}`}
+          schema={schema}
+          errorMessage={errorMessage}
+          autofocus={autofocus}
+          fieldProps={this.props}
```

Comment:
Nice, I like the grouping of these props, makes it clear they're from the json-schema lib 👍